.github/workflows/test.yml

```diff
@@ -0,0 +1,24 @@
+name: Tests
+
+on: [pull_request, push]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+        node: [ '10', '11', '12', '13' ]
+    name: Node ${{ matrix.node }}
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Node
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node }}
+    - name: Install Dependencies
+      run: npm install
+    - name: Run Tests
+      run: npm run test
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Is this needed?

Reply:
Yes it is needed to run, but it is created automatically by GitHub.

Reply:
Are our test requiring the `GITHUB_TOKEN` env?
I check rapidly and it doesn't for me, could you tell us why it is required?

Reply:
@Eomm, https://help.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token#about-the-github_token-secret

Reply:
The question both @Eomm and myself have is why it's needed to run `npm test`. As you see in https://github.com/fastify/fastify/blob/b546ea8aa9335c1816714e03e004f789935f5390/.github/workflows/ci.yml#L42-L46, coveralls action need it. I don't think `npm test` needs it for the tests to pass.

Reply:
Sorry to be "crazy", but if it doesn't need, I would not add it since that token could be read at runtime by our dep and I would do this only with caution in mind